controllers/patch_controllers.js

```diff
@@ -0,0 +1,13 @@
+const {updateReview} = require('../models/patch_models');
+
+exports.patchReview = (req, res, next) => {
+    const { inc_votes } = req.body;
+    const { review_id } = req.params;
+    updateReview(inc_votes, review_id)
+    .then(review => {
+        res.status(200).send(review[0]);
```

Comment:
would we just want to send review[0] or what we want to send a object with a key of review?

Reply:
Thanks Cat. I've amended the patch request output, so it now returns the review object on a key of review. I've also added a test for invalid inputs on patch requests (specifically empty objects). Maybe when I'm further along and I know exactly what users will be allowed to patch, I could change it so the request only goes through if the input object contains one or more of specific properties? Cheers